pom.xml

```diff
@@ -152,72 +191,20 @@
                 </configuration>
             </plugin>
             <plugin>
-	      <!-- used to generate single jar file for deployment -->
                 <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-shade-plugin</artifactId>
```

Comment:
At irregular intervals, `gcloud` was complaining about the size of the jar.  So instead of doing an uberjar, I broke this out into uploads of all dependent jars.